src/protocols/MeterW1therm.cpp

```diff
@@ -83,8 +83,50 @@ bool MeterW1therm::W1sysHWif::readTemp(const std::string &device, double &value)
 			print(log_debug, "CRC not ok from %s (%s)", "w1t", dev.c_str(), buffer);
 		} else {
 			// crc ok
-			// now parse t=<value>
+			// now check for other errors:
```

Comment:
this comment is misplaced...
the result of `fgets()` is not only for error-checking, it's also what the reading is parsed from.